pkg/graph/schema/types/echimp.graphql

```diff
@@ -0,0 +1,45 @@
+type EChimpCR {
+  crNumber: String!
+  versionNum: String!
+  initiator: String
+  firstName: String
+  lastName: String
+  title: String
+  sensitiveFlag: Boolean
+  implementationDate: String
+  crSummary: String
+  crStatus: String
+  emergencyCrFlag: Boolean
+  relatedCrNumbers: String
+  relatedCrTdlNumbers: String
+  associatedModelUids: UUID
+}
+
+type EChimpTDL {
+  tdlNumber: String!
+  versionNum: String!
+  initiator: String
+  firstName: String
+  lastName: String
+  title: String
+  issuedDate: String
+  status: String
+  associatedModelUids: UUID
+}
+
+
+# """
+# EChimpCRAndTDLS is a type that represents CRS and TDLS so they can be returned together in graphql
+# """
+union EChimpCRAndTDLS = EChimpCR  | EChimpTDL
+
+
+
+
+extend type Query {
+  echimpCR(id: String!): EChimpCR!
+  echimpCRs: [EChimpCR!]!
+  echimpTDLs: [EChimpTDL!]!
+  echimpTDL(id: String!): EChimpTDL!
+  echimpCRAndTDLS: [EChimpCRAndTDLS!]!
```

Comment:
I might suggest against exposing CRs and TDLs outside the scope of a model plan until we have a use case for it -- I think the new properties on `type ModelPlan` should suffice

Reply:
We could leave the "fetch by ID" ones in here if that'd be helpful on the FE, though! CC @patrickseguraoddball @garyjzhao

Reply:
Thanks @ClayBenson94 , I'm pretty sure from the FIGMA that there is a need to show all CRs and TDLS, not just ones associated to a model plan (that why I have these routes exposed. Do you see that in the figma as well?

Reply:
I added fetch by ID just to support the possibility of the CR / TDL model appearing as a route (without needing to fetch all the records). It was vaguely talked about as a possibility, so I thought it might be better just to include it

Reply:
> Thanks @ClayBenson94 , I'm pretty sure from the FIGMA that there is a need to show all CRs and TDLS, not just ones associated to a model plan (that why I have these routes exposed. Do you see that in the figma as well?

I wasn't able to find that in the Figma myself, but I might be looking in the wrong spot. This won't really be harmful to introduce to just unblock the FE, but we might want to just reconsider it in the near future if it's not going to be needed!

Reply:
Sounds good! This is the [area](https://www.figma.com/design/qXhO12xSSkDF4LaToGKJoV/MINT-Blue-Sky-Ideas?node-id=2697-35599&node-type=section&t=7fyKzjzrU9gs6tVn-0) I'm thinking of, but it might not need the separate CRs, and separate TDLs endpoint (perhaps just the union type).